### PR TITLE
Add flags on batch command.

### PR DESCRIPTION
**Pull request by mbobrovskyi**
## Fixes / Features
- Add flags on batch command.

## Testing / Documentation
Testing details.

- [ y/n ] Tests pass
- [ y/n ] Appropriate changes to documentation are included in the PR

### CODE REVIEW

**Comment by mbobrovskyi** [comment]
cc @IrvingMg

File: src/xpk/commands/batch.py [hunk 1]
```diff
@@ -46,11 +48,66 @@ def batch(args: Namespace) -> None:
 
 def submit_job(args: Namespace) -> None:
```
Comment (IrvingMg) [inline src/xpk/commands/batch.py:49]: Maybe we're missing `--init-image` or don't we need it?

Comment (IrvingMg) [inline src/xpk/commands/batch.py:49]: Probably we also need to have `--skip-localqueue-validation` and `--skip-priority-validation` flags?

Comment (mbobrovskyi) [reply src/xpk/commands/batch.py:49]: Yeah, this is a question. Do we need it?

Comment (mbobrovskyi) [reply src/xpk/commands/batch.py:49]: WDYT? @44past4 @mwysokin

Comment (mbobrovskyi) [reply src/xpk/commands/batch.py:49]: If localqueue owned by us we don't need to check it. On the other hand user could remove it. So not sure about it to. 

@44past4 @mwysokin @pawloch00 WDYT?

Comment (mbobrovskyi) [reply src/xpk/commands/batch.py:49]: For about priority validation the same question. Do we need it?

@44past4 @mwysokin @pawloch00 WDYT?

Comment (pawloch00) [reply src/xpk/commands/batch.py:49]: There is only one localqueue defined during cluster creation that is used by xpk. I don't think we should allow user to perform any checks of changes on lq.

Comment (pawloch00) [reply src/xpk/commands/batch.py:49]: But lets have @44past4  for final decision

Comment (44past4) [reply src/xpk/commands/batch.py:49]: The main assumption in XPK is that this should be highly opinionated tool with limited number of options available for users. Therefore in this case I would assume that we own the local queue and priorities and users should not edit those in any way.

Comment (mbobrovskyi) [reply src/xpk/commands/batch.py:49]: Thanks!

Comment (44past4) [reply src/xpk/commands/batch.py:49]: I see limited value in the option to override this image when running a workload. Therefore for now I would skip this option.

Comment (mbobrovskyi) [reply src/xpk/commands/batch.py:49]: Thanks!

**Comment by mbobrovskyi** [comment]
Due to switching to local branch.

https://github.com/AI-Hypercomputer/xpk/pull/236

**Comment by mbobrovskyi** [comment]
Due to switching to local branch.

https://github.com/AI-Hypercomputer/xpk/pull/236

**Pull request closed without merging**